docs/source/chatops/aliases.rst

```diff
@@ -166,6 +177,97 @@ The above alias supports the following commands -
     !list sensors from examples
     !list sensors from examples limit=2
 
+"Display-representation" format objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, every format string is exposed in Hubot help as is. This is not always desirable in case you want to make a complicated regex, have ten very similar format strings to "humanize" the input, or hide one of the strings for any other reason.
+
+In this case, instead of having a string in `formats`, you can write an object with a `display` parameter (a string that will show up in help) and a `representation` list (matches that Hubot will actually look for):
+
+.. code-block:: yaml
+
+    formats:
+      - display: "google {{query}}"
+        representation:
+          - "(google|look for) {{query=StackStorm}}[!.]?"
+          - "search google for {{query}}"
+
+This will work as follows:
+
+  - the `display` string (`google {{query}}`) will be exposed via the `!help` command.
+  - strings from the `representation` list (`(google|look for) {{query=StackStorm}}[!.]?` regex, and `search google for {{query}}` string) will be matched by Hubot.
+
+You can use both strings and display-representation objects in `formats` at the same time:
+
+.. code-block:: yaml
+
+    formats:
+      - display: "google {{query}}"
+        representation:
+          - "(google|look for) {{query=StackStorm}}[!.]?"
+          - "search google for {{query}}"
+      - "find me some {{query}}"
+      - "find me some {{query}} in {{engine}}"
+
+Acknowledgement options
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Hubot will acknowledge every ChatOps command with a random message containing StackStorm execution ID and a link to the Web UI. It's possible to customize this message in your alias definition:
+
+.. code-block:: yaml
+
+    ack:
+      format: "acknowledged!"
+      append_url: false
+
+The `format` parameter will customize your message, and the `append_url` flag controls the Web UI link at the end. It is also possible to use Jinja in the format string, with `actionalias` and `execution` comprising the Jinja context:
+
+.. code-block:: yaml
+
+    ack:
+      format: "Executing `{{ actionalias.ref }}`, your ID is `{{ execution.id[:2] }}..{{ execution.id[-2:] }}`"
+
+The `enabled` parameter controls whether the message will be sent. It defaults to `true`, and setting it to `false` will disable the acknowledgement message altogether:
+
+.. code-block:: yaml
+
+    ack:
+      enabled: false
+
+Result options
+^^^^^^^^^^^^^^
+
+Same as with `ack`, you can configure `result` to disable result messages or set a custom format so that Hubot would output a nicely formatted list, filter strings, or switch the message text depending on execution status:
+
+.. code-block:: yaml
+
+    result:
+      format: |
+        {% if execution.result.result|length %}
```

Comment:
I think it would be nice if in the future versions we also supported path to the jinja template files, e.g.:

```yaml
format: "alias/templates/my_result.j2"
```